aicsimageio/readers/ome_zarr_reader.py

```diff
@@ -65,6 +67,8 @@ def __init__(
             )
 
         self._zarr = ZarrReader(parse_url(self._path, mode="r")).zarr
+        self._physical_pixel_sizes: Optional[PhysicalPixelSizes] = None
+        self._multiresolution_level = 0
```

Comment:
Unsure what the plan for this variable is, but for a quick solution to allow this to be user-set you could add it as an optional keyword arg in the constructor for this class

Reply:
Zarr files have a list of coordinate transforms that are associated with the resolution level of the image. In the future, we want to support any of these multi-resolutions using self._multiresolution_level. Dan gave a small hypothetical with 
```
im = AICSImage(my_ome_zarr_path, reader=OmeZarrReader)

nr = im.get_num_resolution_levels()
# now I know there are nr 'datasets' in the metadata for scene 0.

im.set_multiresolution_level(2)

# now if i get physical pixel size it will be different than the default resolution level 0.

# and the shape should come back as the size of the array stored under dataset 2.
im.shape
```
Once we have a working physical pixel size. we can start to work on the resizes and implementation for multi-resolution.




`

Reply:
Just to note, there is a potentially interesting intermediate implementation where the above api (set_multiresolution_level and get_num_resolution_levels) live only in the OmeZarrReader which gets us all the functionality without worrying about adding it to base Reader or AICSImage.  Arguably it's easy to add them to base Reader and just have default implementation that does nothing for all other readers.